### PR TITLE
form input / input

### PR DESCRIPTION
**Pull request by Benedito0**
| - | - |
| --- | --- |
| Issue | ripe-tech/ripe-white#144 |
| Decisions |   |
| Animated GIF |JgsgDTKav8|

### CODE REVIEW

File: vue/components/label-white/label-white.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<template>
```
Comment (3rdvision) [inline vue/components/label-white/label-white.vue]: is there a storybook for this?

File: vue/components/input/input.stories.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        },
+        watch: {
+            value(value) {
+                this.valueData = value;
+            }
+        },
+        template: `
+            <div>
+                    <input-white
+                        v-bind:variant="variant"
```
Comment (3rdvision) [inline vue/components/input/input.stories.js]: Remove prop variant or create the style to be dark and white text inside

File: vue/components/form-input/form-input.stories.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            headerMinWidth: {
+                default: number("Header Minimum Width", null)
+            },
+            footerMinWidth: {
+                default: number("Footer Minimum Width", null)
+            }
+        },
+        template: `
+            <form-input
+                v-bind:variant="variant"
```
Comment (3rdvision) [inline vue/components/form-input/form-input.stories.js]: when variant is inline, the top border gets cutted
imagem

File: vue/components/label/label.stories.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { storiesOf } from "@storybook/vue";
+
+storiesOf("Atoms", module).add("Label", () => ({
+    template: `
+            <div>
```
Comment (joamag) [inline vue/components/label/label.stories.js]: is this indent ok ?

File: vue/components/label.stories.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { storiesOf } from "@storybook/vue";
+
+storiesOf("Atoms", module).add("Label", () => ({
+    template: `
+            <div>
```
Comment (joamag) [inline vue/components/label.stories.js]: indentation...

**Comment by gcandal** [comment]
@Benedito0 please move this to RIPE White using the new component structure.

**Comment by joamag** [comment]
@Benedito0 please use the new atomic structure of ripe white here https://github.com/ripe-tech/ripe-white/tree/master/vue/components input and label are atoms

**Comment by joamag** [comment]
Closing this as we've migrated this to RIPE White.

**Pull request closed without merging**